articles/connections/social/instagram.md

```diff
@@ -6,7 +6,7 @@ image: /media/connections/instagram.png
 seo_alias: instagram
 description: Learn how to add login functionality to your app with Instagram. You will need to generate keys, copy these into your Auth0 settings, and enable the connection.
 toc: true
-public: true
+public: false
```

Comment:
Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.